### PR TITLE
Future-based execution of the BDSS process

### PR DESCRIPTION
**Pull request by stefanoborini**
Reworks the BDSS execution to work in a concurrent future.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,21 +22,24 @@ def write_version_py():
 
 write_version_py()
 
+install_requires = [
```
Comment (martinRenou) [inline setup.py]: Why did you need to export this from the setup arguments?

File: force_wfmanager/wfmanager_task.py [hunk 2]
```diff
@@ -22,15 +22,7 @@
...
-@contextmanager
-def cleanup_garbage(tmpfile):
-    yield
-
-    try:
-        os.remove(tmpfile)
-    except OSError:
-        logging.exception("Could not delete the tmp file {}".format(tmpfile))
-        raise
+log = logging.getLogger(__name__)
```
Comment (martinRenou) [inline force_wfmanager/wfmanager_task.py]: It was already there

Comment (stefanoborini) [inline force_wfmanager/wfmanager_task.py]: Thanks!
